### PR TITLE
Feature/16 create and set routing add method to pollcontroller

### LINKED ISSUE #16: Create and set routing 'add'-Method to PollController
_Before this issue can be solved, #15 must be solved and merged into develop._

To save some polls, we need a http-api endpoint to save some. 

Write a `add`-Method into the `PollController` that creates and returns the created poll as json. The url should be `api/polls/create`. 

Write also some tests for this endpoint. (see below)

The URL should be only accessible with POST-method. (See below)

We don't need a validation / error handling at this moment. It would be a follow-up issue.

The HTTP-Response code should be, if successful, 201. (see below)

Interesting resources:
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods
- https://developer.mozilla.org/en-US/docs/Web/HTTP/Status
- https://laravel.com/docs/9.x/http-tests
- https://laravel.com/docs/9.x/routing
- https://laravel.com/docs/9.x/routing#route-group-prefixes

### PR DESCRIPTION
**Pull request by GrafBunsen**
closing #16

### CODE REVIEW

File: app/Http/Controllers/PollController.php [hunk 1]
```diff
@@ -2,9 +2,26 @@
...
 namespace App\Http\Controllers;
 
+use App\Models\Poll;
 use Illuminate\Http\Request;
+use Illuminate\Http\Response;
 
 class PollController extends Controller
 {
-    //
+    public function create(Request $request): Response
```
Comment (martincodes-de) [inline app/Http/Controllers/PollController.php]: Controller action create doesnt fulfills requirements. see ticket

File: tests/Feature/HttpTest.php [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Tests\TestCase;
+
+
+class HttpTest extends TestCase
+{
+    use RefreshDatabase;
+
+    /**
+     * A basic feature test example.
```
Comment (martincodes-de) [inline tests/Feature/HttpTest.php]: edit or remove description

File: tests/Feature/HttpTest.php [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+     *
+     * @return void
+     */
+    public function test_example(): void
```
Comment (martincodes-de) [inline tests/Feature/HttpTest.php]: rename method

**Review by martincodes-de — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/PollController.php (hunk 1 above)
Comment (GrafBunsen) [reply app/Http/Controllers/PollController.php]: done

File: tests/Feature/HttpTest.php (hunk 2 above)
Comment (GrafBunsen) [reply tests/Feature/HttpTest.php]: removed

File: tests/Feature/HttpTest.php (hunk 3 above)
Comment (GrafBunsen) [reply tests/Feature/HttpTest.php]: renamed

File: app/Http/Controllers/PollController.php [hunk 4]
```diff
@@ -2,9 +2,26 @@
...
+        $phase = $request->input('phase');
+
+        $poll = new Poll();
+        $poll->name = $name;
+        $poll->description = $description;
+        $poll->end_date = $endDate;
+        $poll->phase = $phase;
+
+        $poll->save();
+        return response()->json($poll,status: 201);
```
Comment (martincodes-de) [inline app/Http/Controllers/PollController.php]:
```suggestion
        return response()->json($poll, status: 201);
```

**Review by martincodes-de — CHANGES_REQUESTED** [review]

**Review by martincodes-de — APPROVED** [review]

**Pull request merged**